packages/cli-kit/src/public/node/ui.tsx

```diff
@@ -235,3 +236,10 @@ export async function renderConfirmation({question, infoTable}: ConfirmationProp
 
   return prompt({message: question, choices, infoTable})
 }
+
+/**
+ * Runs async tasks and displays their progress to the console.
+ */
+export function renderTasks(tasks: Task[]) {
```

Comment:
Task is a `public` interface so I recommend moving it to the `public/node` folder as well. Otherwise we might introduce breaking changes there in the future without realising it.

Reply:
`renderPrompt` also uses `PromptProps` which is an interface defined privately. We could remove the dependency on these private interfaces by creating new public ones that mirror the private ones, but this might introduce some differences in the future that typescript will not catch for us (e.g. we add a field to the private ones and forget to update the public one).